subm/src/subm.js

```diff
@@ -50,6 +51,7 @@ const Site = freeze({
   badLoginPath: '/loginRefused',
   uploadSlogPath: '/participant/slogForm',
   uploadGenTxPath: '/participant/gentxForm',
+  loadGenKeyPath: '/participant/loadGenKey',
```

Comment:
You're arguably more right here with capitalization, but in the `testnet-load-generator` we seem to use `loadgen` all lower case..